### PR TITLE
Divide home page categories. Link CI category to CET->Extension->Comm…

### PR DESCRIPTION
**Pull request by yewch**
Divide home page categories into Consumer Product and Developer Products. 
Quick link the Commercial Interior category into the Commercial Interior section under CET->Extension->Commercial Interior...

### CODE REVIEW

File: templates/home_page.hbs [hunk 1]
```diff
@@ -17,24 +17,38 @@
   <div class="container">
     <section class="section knowledge-base">
       <section class="categories blocks">
+      <h1 class="geo-medium" style="font-size: 40px">Consumer Products</h1>
```
Comment (hdm-hmnk) [inline templates/home_page.hbs]: Maybe `Consumer Products` should be in the list of translations. Could be added as Dynamic Content.

File: templates/home_page.hbs [hunk 2]
```diff
@@ -50,6 +64,47 @@
...
+          </li>
+          {{/isnt}}
+          {{/isnt}}
+          {{/isnt}}
+          {{/isnt}}
+          {{/isnt}}
+          {{/isnt}}
+          {{/each}}
+        </ul>
+        <h1 class="geo-medium" style="font-size: 40px">Developer Products</h1>
```
Comment (hdm-hmnk) [inline templates/home_page.hbs]: Translations for this as well.

File: templates/home_page.hbs (hunk 1 above)
Comment (yewch) [reply templates/home_page.hbs]: Got it, I'll put it as {{dc 'consumer_products'}} for now.

File: templates/home_page.hbs (hunk 2 above)
Comment (yewch) [reply templates/home_page.hbs]: Placed this as {{dc 'developer_products'}} for now.

**Pull request merged**